packages/buckets/package.json

```diff
@@ -30,16 +30,16 @@
   "license": "MIT",
   "dependencies": {
     "@improbable-eng/grpc-web": "^0.13.0",
-    "@textile/buckets-grpc": "2.1.7",
     "@textile/context": "^0.9.2",
     "@textile/crypto": "^1.0.1",
-    "@textile/grpc-authentication": "^1.0.1",
-    "@textile/grpc-connection": "^1.0.1",
-    "@textile/grpc-transport": "^0.1.0",
+    "@textile/buckets-grpc": "2.1.7",
+    "@textile/grpc-authentication": "^1.0.2",
+    "@textile/grpc-connection": "^1.0.2",
+    "@textile/grpc-transport": "^0.2.1",
     "@textile/hub-grpc": "2.1.7",
-    "@textile/hub-threads-client": "^2.0.1",
+    "@textile/hub-threads-client": "^2.0.2",
```

Comment:
Glad you caught this. I have no idea why this _isn't_ updating in master? But the released version is...